src/components/PersonLinks/PersonLinks.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLinks: React.FC<Props> = ({ person, people }) => {
+  const { slug: id } = useParams();
```

Comment:
The `useParams` hook is used to extract the `slug` parameter from the URL. Ensure that the `slug` parameter is correctly defined in the route configuration to avoid potential undefined values.